app/assets/stylesheets/govuk_publishing_components/components/_organisation-logo.scss

```diff
@@ -81,80 +72,62 @@
   }
 }
 
-@mixin crest($crest) {
-  background: url("govuk_publishing_components/crests/#{$crest}_13px.png") no-repeat 5px 0;
```

Comment:
Looks like we're no longer using crest images with the suffixes `_13px.png`, `_13px_x2.png` and `_18px.png`. Can these files be safely removed?

Reply:
@andysellick I did consider that but decided to be cautious just in case these are used anywhere else that is not covered by this PR. My thinking is that when these logos are replaced with SVGs, we can do a more thorough job on the cleanup.

Reply:
After reviewing other repos, there's confidence that we don't need the redundant images so they've been removed.